src/cplscheme/config/CouplingSchemeConfiguration.cpp

```diff
@@ -697,6 +697,10 @@ void CouplingSchemeConfiguration::addRelativeConvergenceMeasure(
                 "Please check the <relative-convergence-measure limit=\"{}\" data=\"{}\" mesh=\"{}\" /> subtag "
                 "in your <coupling-scheme ... /> in the preCICE configuration file.",
                 limit, dataName, meshName);
+  if (math::smallerEquals(limit / 10, 0.0)) {
+    PRECICE_WARN("Relative convergence limit is too small. This may cause a conflict with "
+                 "the quasi-Newton acceleration scheme if selected. ");
```

Comment:
"You chose a relative convergence limit of X. This is close to the hard-coded numerical resolution of preCICE, Y, which might lead to numerical stability issues."
... or similar.
It's not only the quasi-Newton acc, we could also get problems in the mapping etc. And we need the numbers here to prevent the user from starting a trial-and-error testing.

Using `smallerEquals` is problematic here as soon as the check above is removed. Let's better check the absolute value of the limit.

Reply:
Are we planning on removing the check here?

Reply:
Something like 
```
 if (limit > 10*math::NUMERICAL_ZERO_DIFFERENCE){
    PRECICE_WARN("The relative convergence limit=\"{}\" is close to the hard-coded numerical resolution of preCICE. "
                 "This may lead to instabilities. Increase the relative convergence limit. ", limit);
  }
  ```

Reply:
Why not simply check `math::equals(limit, 0)`?
If this is true, then the limit is too small.

Reply:
We should also give the user the value of `math::NUMERICAL_ZERO_DIFFERENCE` to avoid trial-and-error testing.